bee-common/bee-packable/src/packer/vec.rs

```diff
@@ -18,11 +18,21 @@ impl VecPacker {
         Self::default()
     }
 
+    /// Create an empty packer with an initial capacity.
+    pub fn with_capacity(capacity: usize) -> Self {
+        Self(Vec::with_capacity(capacity))
+    }
+
     /// Use the backing `Vec<u8>` to create an `Unpacker`.
     pub fn as_slice(&self) -> SliceUnpacker<'_> {
         SliceUnpacker::new(self.0.as_slice())
     }
 
+    /// Consumes the `VecPacker` and returns the inner `Vec<u8>`.
```

Comment:
Sometime 3rd person, sometime not, align!

Reply:
:trollface: